README.md

```diff
@@ -1,46 +1,194 @@
+<details>
+    <summary>Client</summary>
+    <ul>
+      <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+      <li><a href="https://react.dev/">React</a></li>
+      <li><a href="https://redux-toolkit.js.org/">Redux</a></li>
+      <li><a href="https://testing-library.com/docs/react-testing-library/intro/">React-Testing-Library</a></li>
+      <li><a href="https://reactrouter.com/en/main">React-Router</a></li>
+      <li><a href="https://www.w3schools.com/js/js_es6.asp">ES6</a></li>
+    </ul>
+  </details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage">LocalStorage</a></li>
+   <li><a href="https://rapidapi.com/digiwalls/api/free-to-play-games-database">FreeToGame API using RapidApi</a></li>
+  </ul>
+</details>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<!-- Features -->
 
-### `npm run eject`
+### Key Features <a name="key-features"></a>
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+  - **[Mobile App]** Designed as a mobile application, offering accessibility and convenience for users on the go.
+  - **[Interactive]** Provides an engaging and interactive user experience, enhancing user interaction and immersion.
+  - **[User-Friendly]** Prioritizes ease of use and intuitive design, ensuring a friendly experience for all users.
+  - **[Reliable]** Built with robustness in mind, delivering reliable performance and stability for seamless usage.
+  - **[Efficient]** Optimized for efficiency, maximizing resource utilization and delivering swift response times.
+  - **[Tested]** Thoroughly tested using industry-standard testing methodologies, ensuring high quality and reliability.
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo & preview <a name="live-demo"></a>
+https://www.loom.com/share/928fd44ee8024d2ba9e7f4fd0ec1944b
```

Comment:
- [ ] awesome video, I truly enjoyed your video presentation, you have good English and a proper flow of information. But there is a bit issue - > you did not show a piece of code that you are proud of as required by the project requirements. Kindly ensure you show a section of your programs, in details.

Reply:
@bobb-Rob  Thank you!! I didn't notice that part 😅 already recorded and updated!